app/assets/javascripts/components/overview/overview_stats.jsx

```diff
@@ -108,96 +108,67 @@ const OverviewStats = ({ course }) => {
   }
 
   let wikidataStats;
+
   if (course.course_stats) {
     const statistics = course.course_stats.stats_hash['www.wikidata.org'];
+    const statsArray = [
+      { 'items created': statistics['items created'], 'items cleared': statistics['items cleared'] },
+      { 'claims created': statistics['claims created'], 'claims changed': statistics['claims changed'], 'claims removed': statistics['claims removed'] },
+      { 'aliases added': statistics['aliases added'], 'aliases changed': statistics['aliases changed'], 'aliases removed': statistics['aliases removed'] },
+      { 'descriptions added': statistics['descriptions added'], 'descriptions changed': statistics['descriptions changed'], 'descriptions removed': statistics['descriptions removed'] },
+      { 'interwiki links added': statistics['interwiki links added'], 'interwiki links removed': statistics['interwiki links removed'] },
+      { 'labels added': statistics['labels added'], 'labels changed': statistics['labels changed'], 'labels removed': statistics['labels removed'] },
+      { 'merged from': statistics['merged from'], 'merged to': statistics['merged to'], },
+      {
+        'other updates': statistics['other updates'],
+        'qualifiers added': statistics['qualifiers added'],
+        'redirects created': statistics['redirects created'],
+        'restorations performed': statistics['restorations performed'],
+        'reverts performed': statistics['reverts performed'],
+        'no data': statistics['no data'] }
+    ];
+
+    statsArray.forEach((stat) => {
+      Object.keys(stat).forEach((key) => {
+        if (stat[key] === 0) {
+          delete stat[key];
+        }
+      });
+    });
+
+    wikidataStats = statsArray.map((stat) => {
+      if (Object.keys(stat).length === 1) {
+        const label = Object.keys(stat)[0].split(' ').join('_');
+        return <OverviewStat
+          key={label}
+          id={(Object.keys(stat)[0].split(' ').join('-'))}
+          className={valueClass(label)}
+          stat={Object.values(stat)[0]}
+          statMsg={I18n.t(`metrics.${label}`)}
+        />;
+      }
+      if (Object.keys(stat).length > 1) {
+        const infoStats = [];
+        for (let i = 1; i < Object.keys(stat).length; i += 1) {
+          infoStats.push([Object.values(stat)[i], I18n.t(`metrics.${Object.keys(stat)[i].split(' ').join('_')}`)]);
+        }
+        const label = Object.keys(stat)[0].split(' ').join('_');
+
+        return <OverviewStat
+          key={label}
+          id={Object.keys(stat)[0].split(' ').join('-')}
+          className={valueClass(label)}
+          stat={Object.values(stat)[0]}
+          statMsg={I18n.t(`metrics.${label}`)}
+          info={infoStats}
+          infoId={`${label}-info`}
+        />;
+      }
+      return <div />;
```

Comment:
This is why there are empty slots for the zero-value stats? I think it shouldn't leave empty space for these. That's similar to how the top-level stats already behave; only 4 stats are shown when a course doesn't have any activity yet (users, articles edited, words added, edit count), while others like upload count are only rendered if nonzero, and the overview stats component renders them evenly spread out no matter how many there are.